bash_scripting_101/bash_scripting_101.md

```diff
@@ -92,21 +107,21 @@ Everything that is necessary to run bash on your Mac or Windows machine is proba
 - Search Bash in Cortana to launch WSL for the first time and click “Y” when asked to install Ubuntu for the first time. [Microsoft has their own internal documentation on WSL which features additional visual resources and FAQS](https://docs.microsoft.com/en-us/windows/wsl/).
 - Create a username and password when prompted and now you are ready to do some shell scripting! These user credentials have no bearing on your Windows user name and you will be automatically logged in for future sessions.
 
-### Knowledge Check 1
+### Quiz: Accessing the bash shell
 
 What is the name of the pre-installed program used to run shell scripts on a Mac?
 
 [( )] Together
 [( )] MSL (Mac Subsystem for Linux)
 [(X)] Terminal
 [( )] Ubuntu
-***********************************************************************
+***
 <div class = "answer">
 
 WSL is the Windows utility for running shell. Ubuntu is a well known Linux operating system that is often used as the default shell scripting environment.
 
 </div>
-***********************************************************************
+***
 
 ## Setting up a User/Home Directory
```

Comment:
I'm not sure if this section is supposed to walk me through doing something, or show me where to find something that already exists. If a user doesn't have CLI experience, how would they find out if they are in their home directory or navigate to it? 

Maybe this page could contain instructions on how to navigate using `cd`, as well as `ls` and `pwd`